src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,32 @@
-export const TodoInfo = () => {};
+import cn from 'classnames';
+import { UserInfo } from '../UserInfo';
+
+interface User {
+  name: string;
+  email: string;
+}
+
+interface Todo {
+  [x: string]: unknown;
```

Comment:
The index signature `[x: string]: unknown;` in the `Todo` interface is unnecessary unless you expect dynamic properties. If not needed, consider removing it to avoid potential misuse or confusion.